### ISSUE TITLE
With pandoc installed works the .docx report of the 3.3 version but  pandoc is not available for 32 bits windows and the report generation fails 

### ISSUE THREAD

**Issue by jorgeegm** [biodosetools-team/biodosetools#21]
### Technical details

- Version:
- Assessment:
- Module:

### Issue description

Describe what issue or problem you are experiencing on the application.

**Comment by aldomann** [comment]
Si descargas el codigo fuente desde la sección de Releases (https://github.com/biodosetools-team/biodosetools/archive/v3.4.0.tar.gz), puedes instalar Biodose Tools ejecutando el siguiente comando en la consola:

```r
install.packages("<path>/biodosetools-3.4.0.tar.gz", repos = NULL, type = "source")
```
donde `<path>` es el directorio donde se ha descargado `.tar.gz`.

**Comment by jorgeegm** [comment]
I closed the issue installing version 2.9 of pandoc the latest available for 32 bits Windows did not solve the report failure.
